foundations/azure/referenceImplementations/core/managementGroupTemplates/policyDefinitions/Compliant-CorpLzPolicySetDefinition.json

```diff
@@ -10,7 +10,133 @@
+                                    },
+                                    "deployment": {
+                                        "properties": {
+                                            "mode": "incremental",
+                                            "template": {
+                                                "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+                                                "contentVersion": "1.0.0.0",
+                                                "parameters": {
+                                                    "privateDnsZoneId": {
+                                                        "type": "string"
+                                                    },
+                                                    "privateEndpointName": {
+                                                        "type": "string"
+                                                    },
+                                                    "location": {
+                                                        "type": "string"
+                                                    }
+                                                },
+                                                "resources": [
+                                                    {
+                                                        "name": "[concat(parameters('privateEndpointName'), '/deployedByPolicy')]",
+                                                        "type": "Microsoft.Network/privateEndpoints/privateDnsZoneGroups",
+                                                        "apiVersion": "2020-03-01",
+                                                        "location": "[parameters('location')]",
+                                                        "properties": {
+                                                            "privateDnsZoneConfigs": [
+                                                                {
+                                                                    "name": "privateDnsZone",
+                                                                    "properties": {
+                                                                        "privateDnsZoneId": "[parameters('privateDnsZoneId')]"
+                                                                    }
+                                                                }
+                                                            ]
+                                                        }
+                                                    }
+                                                ]
+                                            },
+                                            "parameters": {
+                                                "privateDnsZoneId": {
+                                                    "value": "[parameters('privateDnsZoneId')]"
```

Comment:
Please escape the ARM template functions properly using this pattern "[[parameter('paramName')]" if not, it is not deployable.